grails-app/controllers/org/pih/warehouse/api/InventoryItemApiController.groovy

```diff
@@ -0,0 +1,30 @@
+/**
+* Copyright (c) 2012 Partners In Health.  All rights reserved.
+* The use and distribution terms for this software are covered by the
+* Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+* which can be found in the file epl-v10.html at the root of this distribution.
+* By using this software in any fashion, you are agreeing to be bound by
+* the terms of this license.
+* You must not remove this notice, or any other, from this software.
+**/
+package org.pih.warehouse.api
+
+import org.codehaus.groovy.grails.web.json.JSONObject
+import org.pih.warehouse.inventory.InventoryService
+
+class InventoryItemApiController {
+
+    InventoryService inventoryService
+
+    def validateLot = {
+        JSONObject jsonObject = request.JSON
+        Boolean areLotsValid = inventoryService.areLotsValid(jsonObject?.items)
+
+        if (!areLotsValid) {
+            render(status: 400)
```

Comment:
Should we have an error response? This is going to be part of our API so it should be well-designed and work intuitively for all use cases, not just this very specific feature.